pkg/cache/v3/simple.go

```diff
@@ -496,6 +505,15 @@ func (cache *snapshotCache) respond(ctx context.Context, request *Request, value
 	}
 }
 
+func getForcePushEdsResourcesNames(typ types.ResponseType, response *RawDeltaResponse) ([]string, types.ResponseType, bool) {
```

Comment:
nitpick: (subjective) the name is a bit weird

```suggestion
func getEdsResourceNamesToForcePush(typ types.ResponseType, response *RawDeltaResponse) ([]string, types.ResponseType, bool) {
```